docs/source/roadmap.md

```diff
@@ -147,23 +76,20 @@ Nevertheless, to demonstrate _starfish_'s scale, we will implement workflows on
 ### A Developer Focus Point
```

Comment:
We should rephrase this a bit. The key point here is to help SpaceTx benchmark and compare their data. _Additionally_, we want starfish to be a developer focus and to accomplish the goals stated in this paragraph.

Reply:
Agreed. This whole section feels weird to me when it switches from a longform description of the work to to-do lists. Let's do this in your branch after the merge.